docs/components/content/ComponentCard.vue

```diff
@@ -206,7 +207,7 @@ const code = computed(() => {
       code += `>
   ${props.code}</${name}>`
     } else {
-      code += `>${props.code}</${name}>`
+      code += `>${props.code} </${name}>`
```

Comment:
Why add a space here? It causes this: https://ui-7s1fnh27q-nuxt-js.vercel.app/dev/elements/badge#usage

Reply:
I tried to remove it but then it breaks the `form-group` examples here: https://ui-7s1fnh27q-nuxt-js.vercel.app/dev/forms/form-group#help

Reply:
@KeJunMao Any idea on how to fix this?

Reply:
what do you think about this: 

```diff
- code += `>${props.code} </${name}>`
+ code += `>${props.code.endsWith('>') ? \`${props.code}\n` : props.code}</${name}>`
```

Reply:
Thanks 😊